build-gradle-plugin/build.gradle

```diff
@@ -30,3 +31,17 @@ ext {
 
 apply from: rootProject.file('gradle/bintray/install.gradle')
 apply from: rootProject.file('gradle/bintray/bintray.gradle')
+
+// This allows you to publish to local maven repository on your machine via publishToMavenLocal task
+// for testing purposes. You can then reference this plugin the standard way in your project but it
+// will be fetched from this local maven repository.
+publishing {
```

Comment:
Can you add mention about testing with local maven to Readme pls? if its not already there

Reply:
Fixed in cb4aaa7c649aa4627fb93cd2c55f800ac40463ae